ui/components/app/signature-request/signature-request.component.js

```diff
@@ -34,10 +33,8 @@ export default class SignatureRequest extends PureComponent {
     }
   }
 
-  _beforeUnload = (event) => {
+  _beforeUnload = (_) => {
```

Comment:
Nit: This argument can be omitted entirely since we're not using it

```suggestion
  _beforeUnload = () => {
```